kolibri/content/serializers.py

```diff
@@ -32,78 +30,53 @@ def to_representation(self, value):
         return self.reverse(view_name, kwargs=kwargs, request=request, format=format)
 
 
-class ContentMetadataSerializer(serializers.ModelSerializer):
+class ContentNodeSerializer(serializers.ModelSerializer):
     url = DualLookuplinkedIdentityField(
-        view_name='contentmetadata-detail',
+        view_name='contentnode-detail',
         lookup_field_1='channelmetadata_channel_id',
```

Comment:
You need to rename these `lookup_field`s as well, to remove mentions of `channelmetadata`.